auth/requirements.txt

```diff
@@ -1,5 +1,6 @@
-aiohttp>=3.3.0,<4.0.0
+aiohttp>=3.7.3,<4.0.0
 backoff>=1.0.0,<2.0.0
+chardet>=2.0,<4.0  # required by aiohttp 3.7.3
```

Comment:
Seems fine. Just curious, do we know of clients who would want `chardet==2.x`? Do we know aiohttp 3.7.3 works okay with chardet 2.x?

Reply:
I don't know what our users want TBH -- but, chardet>=2.0 lower bound is actually copied over from aiohttp 3.7.3 's requirements.

Reply:
Okay then that's fine I guess. Thanks for confirming!